app/graphql/types/query_type.rb

```diff
@@ -101,8 +101,10 @@ def crowd_agg_buckets(search_hash: nil, params: nil, url: nil, config_type: nil,
       )
     end
 
-    def crowd_agg_facets(search_hash: nil, params: nil)
+    def crowd_agg_facets(search_hash: nil, params: nil, crowd_buckets_wanted: nil)
```

Comment:
what is the "params" value in this line doing or is it the prior item that was commented out that was previously commented out?

Reply:
It seems to be the part that was commented out. If you look father up in the file the params were commented out as a argument for the crowd_agg_facet query. This method would always just set params to nil because would not get the value from anywhere else as far as I am aware. It also does not seem to get the search_hash either. Essentially the way it was defined it takes no arguments but here it has some arguments defaulted.